app/scripts/views/mixins/signin-mixin.js

```diff
@@ -96,6 +96,41 @@ define(function (require, exports, module) {
 
       return this.invokeBrokerMethod(brokerMethod, account)
         .then(this.navigate.bind(this, this.model.get('redirectTo') || 'settings', {}, navigateData));
+    },
+
+    /*anchor tag present in both signin and signup views*/
+    events: {
+      'click #suggest-sync a': 'onSuggestSyncClick'
+    },
+
+    /**
+     * Sets showSyncSuggestion var in view context (signin and signup)
+     * sends log information about suggestion panel visibility
+     * @method setShowSyncSuggestion
+     * @param {object} context: view context object
+    */
+
+    setShowSyncSuggestion: function (context) {
+      if (! this.relier.get('service') && ! context.isSyncMigration && ! context.isAmoMigration) {
```

Comment:
>  ! context.isSyncMigration && ! context.isAmoMigration

no need to check for that, we can show 2 infos worst case, but should not happen anyway